src/components/shapes/TodoListShape.js

```diff
@@ -0,0 +1,6 @@
+import PropTypes from 'prop-types';
+import { TodoShape } from './TodoShape';
+
+export const TodoListShape = PropTypes.shape({
+  tasks: PropTypes.arrayOf(TodoShape).isRequired,
+}).isRequired;
```

Comment:
Тут можна і звичайний об'єкт
```suggestion
export const TodoListShape = {
  tasks: PropTypes.arrayOf(TodoShape).isRequired,
}
```

Reply:
взагалі не розумію коли як...

Reply:
Вже після Юриного розбору трохи ясніше але прояснення ще в процесі під час практики))